assets/css/abstracts/_mixins.scss

```diff
@@ -75,3 +75,18 @@
 	margin: unset;
 	overflow: hidden;
 }
+
+// Reset <button> style so we can use link style for action buttons.
+@mixin link-button() {
```

Comment:
Nice! This mixin will be handy in many ocasions. :clap: 

I wonder if it should be named `button-reset` instead. Technically it could be used in other occasions not only on link buttons.

Reply:
Works for me, though I'd probably tend to naming it based on a specific intention.